style.css

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  width: 375px;
+  height: 718px;
+  padding: 0 24px;
+}
+
+.hero {
+  background-image: url(./imges/background\ image.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position-x: left;
+}
```

Comment:
- [ ] [OPTIONAL] As mentioned in the project requirements **In order to lay out the elements on the page you should use flexbox in all 2 sections.** you should try to use flex for the layout of this section currently you have used it only for the header section, flexbox is very crucial in responsive web design and is an important skill as a front-end dev, please try to use it as much as possible for a better responsive design :+1: